.goreleaser.yml

```diff
@@ -133,3 +133,4 @@ signs:
   -
     id: gopass
     artifacts: checksum
+    args: ["-u", "0x21FF4D8F35D1AEE5", "--armor", "--output", "${signature}", "--detach-sign", "${artifact}"]
```

Comment:
Maybe use the "--batch" args here? I believe it is passed to `gpg`

Reply:
Shouldn't be necessary: https://goreleaser.com/customization/sign/